### PR TITLE
Desktop Portfolio Project

### PR DESCRIPTION
**Pull request by Carshy**
In this project, I created a desktop version of my portfolio website. I used several tools and technologies such as Figma design to meet the requirements. During the project, I learned the following:
* How to use media queries to create the desktop version of a website
* How to use Figma to create a desktop version
* Using CSS3 to align, position, and style HTML elements
* Setting up backgrounds images

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -599,3 +691,500 @@ form .contacts-lists {
...
+    width: 920px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    margin-left: 18.056%;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    margin-top: 21%;
```
Comment (omarsalem7) [inline styles/style.css:808]: - [ ] Kindly remove margin-top to make your headline in center of screen not bottom like Figma design.

File: index.html [hunk 2]
```diff
@@ -44,75 +57,87 @@ <h2 class="connect"><a href="#">LET’S CONNECT</a></h2>
...
+                            <ul class="languages">
+                                <li class="html-language">html</li>
+                                <li class="css-language">css</li>
+                                <li class="JavaScript-language">javaScript</li>
+                            </ul>
+                            <div class="project-btn">
+                                <button class="details-button" type="button">See Project</button>
+                            </div>  
+                        </div>
+                </div>
```
Comment (omarsalem7) [inline index.html:79]: - [ ] Please consider making the work project in canter of screen to match Figma template.


| Your design  | Figma design|
| ------------- | ------------- |
| image | image
 |

File: styles/style.css [hunk 3]
```diff
@@ -599,3 +691,500 @@ form .contacts-lists {
...
+  .connect1 a {
+    width: 117px;
+    height: 24px;
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    color: #7f8cff;
+    text-decoration: none;
+    margin-left: 24px;
```
Comment (omarsalem7) [inline styles/style.css:745]: - [ ] Kindly remove margin left to match  Figma design

File: index.html [hunk 4]
```diff
@@ -121,53 +146,60 @@ <h3 class="tonic">Multi-Post Stories</h3>
...
+                    <ul class="languages-list">
+                        <li><img class="js-logo" src="images/css.png" alt="javaScript"></li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+
+                <ul class="languages-p">
+                    <li>Frameworks</li>
+                    <li><img class="arrow-pointers" src="images/arrow.png" alt="dropdown"><li>
+                </ul>
```
Comment (omarsalem7) [inline index.html:194]: - [ ] Please try to match Figma design by align arrow for framework in right 

| Your design  | Figma design|
| ------------- | ------------- |
| image | image|

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Carshy, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Carshy** [comment]
Hello @omarsalem7 Changes made as requested. I also don't understand what happened to the branch I see it indicating merge. what happened is that I was submitting a previous assignment on the main branch that required merging after reviewing it individually. Going to my project branch I have seen it indicating as merged and I have tried different  ways to restore it but to no success. Kindly find the changes made as requested. I have also tried to center my webpage but it is not working. We learn how to do that Thank you

**Pull request merged**